joint_message[22] joints
float32 tempo
string moveName
int32 currentFrame